examples/basic/main.tf

```diff
@@ -1,5 +1,9 @@
 provider "azurerm" {
   features{}
+  skip_provider_registration = true

```

Comment:
I wouldn't put this in the example. This is only needed for our test environment and not relevant for most users